Makefile

```diff
@@ -80,6 +81,15 @@ build:
 	@[ "${GIT_HASH}" ] || ( echo ">> GIT_HASH is not set"; exit 1 )
 	go build -ldflags "-X main.GitTag=$(GIT_TAG) -X main.GitHash=$(GIT_HASH) -s -w" ./cmd/telliot
 
+.PHONY: generate-config-docs
+generate-config-docs: ## Use the example env file and the actual golang config struct to generate the setup doc page. .... <- add more info how it works.
+	@go run ./scripts/cfgdocgen --output docs/configuration.md
+
+.PHONY: check-config-docs
+check-config-docs:
+	@go run ./scripts/cfgdocgen --output tmp/configuration.md
+	"$(CMP)" --silent docs/configuration.md tmp/configuration.md || (echo >&2 "outdated docs/configuration.md, run the config docs generator script" ; exit 1)
```

Comment:
why not use what linting does just by using git, this way will reduce the dependancies.